LanguageFeatures/FinalizationRegistry/ffi/Finalizer/throw_A01_t01.dart

```diff
@@ -25,13 +25,14 @@ final Finalizer finalizer = Finalizer((token) {
 @pragma('vm:never-inline')
 void attachToFinalizer() {
   var o = Object();
-  finalizer.attach(o, "throw");
+  // This would cause the program to exit with `Unhandled exception`.
```

Comment:
See the description of the test: "Checks that all exceptions thrown by finalization callbacks are ignored".
It seems like if program exits with an unhandled exception then it's an implementation bug.

Reply:
The specification says: "Finalization callbacks must not throw." It does not specify that callbacks are ignored.

I'll update the test description.

Reply:
Or delete the test, because there's nothing left to test if we don't check that Exceptions are ignored.

@lrhn any suggestions?